### PR TITLE
Karen/conversion

### PR DESCRIPTION
**Pull request by dythya**

### CODE REVIEW

File: SimPy/stepping.py [hunk 1]
```diff
@@ -31,7 +33,7 @@ def stepping(glob):
         while True:
             if sys.version_info.major == 2:
                 input = raw_input
-            cmd = input("Command ('h' for help): ")
+            cmd = eval(input("Command ('h' for help): "))
```
Comment (rhysmdnz) [inline SimPy/stepping.py:36]: Does this need to be evaluated?

File: SimPy/SimulationGUIDebug.py [hunk 2]
```diff
@@ -166,7 +170,7 @@ def promptUser():
     while( 1 ):
         if sys.version_info.major == 2:
             input = raw_input
-        user_input = input( prompt )
+        user_input = eval(input( prompt ))
```
Comment (rhysmdnz) [inline SimPy/SimulationGUIDebug.py:173]: Again do we need an eval here?

**Review by rhysmdnz — COMMENTED** [review]
There are lots of uses of the past library throughout this code. How many of these do we actually need?

**Comment by dythya** [comment]
Closing - we don't need to run futurize as code already runs on Python 3

**Pull request closed without merging**